docs/CONFIG-VARS.md

```diff
@@ -204,6 +204,7 @@ When setting V4_CFG_TLS_MODE to a value other than "disabled" and no V4_CFG_TLS_
 | V4_CFG_POSTGRES_FQDN | Existing postgres ip/fqdn | string | | true | | viya |
 | V4_CFG_POSTGRES_PORT | Existing postgres port | string | 5432 | false | | viya |
 | V4_CFG_POSTGRES_DATABASE | Existing postgres database name | string | "SharedServices" | false | | viya |
+| V4_CFG_POSTGRES_SSL_ENFORCEMENT | Require ssl connection to existing postgres | bool | false | false | | viya |
```

Comment:
@NormJohnIV  I don't see the change we discussed with regard to a GCP warning.

Reply:
updated